Cannot import my database backup

## Overview 
Hi,
I can't import my database backup, either through the docker-compose up method (when adding file in project) or though the wp cli command. 


## Technical informations 
OS : 
- What is your docker version? 
```
MacBook-Air-de-Laila:wordpress-starter leiluspocus$ docker version
Client:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   6b644ec
 Built:        Wed Oct 26 23:26:11 2016
 OS/Arch:      darwin/amd64

Server:
 Version:      1.12.3
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   6b644ec
 Built:        Wed Oct 26 23:26:11 2016
 OS/Arch:      linux/amd64
```

## `docker-compose.yml`

```yml
version: '2'
services:
  wordpress:
    image: visiblevc/wordpress:latest
    ports:
      - 8080:80
      - 443:443
    volumes:
      - data:/data
      - ./braxton:/app/wp-content/themes/braxton   # Theme development
    environment:
      DB_NAME: mywebsite
      DB_PASS: root      
      DB_PREFIX: mwb_
      THEMES: >-
        [local]braxton
      SEARCH_REPLACE: local.mywebsite.com,localhost:8080
  db:
    image: mysql:5.7 # or mysql:5.7
    volumes:
      - data:/var/lib/mysql
    environment:
      MYSQL_ROOT_PASSWORD: root
volumes:
  data: {}
```

## Project structure

```
MacBook-Air-de-Laila:wordpress-starter leiluspocus$ ls -l
total 104
-rw-r--r--   1 leiluspocus  staff   2001 12 déc 09:38 CHANGELOG.md
-rw-r--r--   1 leiluspocus  staff   1555 12 déc 09:38 ISSUE_TEMPLATE.md
-rw-r--r--   1 leiluspocus  staff   8129 12 déc 09:38 README.md
lrwxr-xr-x   1 leiluspocus  staff     13 12 déc 09:41 braxton -> ../braxton-d/
drwxr-xr-x   4 leiluspocus  staff    136 12 déc 09:41 data
-rw-r--r--   1 leiluspocus  staff    565 12 déc 09:50 docker-compose.yml
-rw-r--r--   1 leiluspocus  staff   5119 12 déc 09:38 gulpfile.js
drwxr-xr-x  27 leiluspocus  staff    918 12 déc 09:40 node_modules
-rw-r--r--   1 leiluspocus  staff   2478 12 déc 10:04 npm-debug.log
-rw-r--r--   1 leiluspocus  staff   1120 12 déc 09:38 package.json
drwxr-xr-x   3 leiluspocus  staff    102 12 déc 09:38 php5.6
drwxr-xr-x   3 leiluspocus  staff    102 12 déc 09:38 php7.0
-rw-r--r--   1 leiluspocus  staff  11494 12 déc 09:38 run.sh
```

Here is also the content of the data folder : 
```
MacBook-Air-de-Laila:wordpress-starter leiluspocus$ ls -l data
total 4244544
-rwxr-xr-x@ 1 leiluspocus  staff  2173205465 12 déc 09:42 database.sql
```

## `docker-compose up` output

```
MacBook-Air-de-Laila:wordpress-starter leiluspocus$ docker-compose up
Creating network "wordpressstarter_default" with the default driver
Creating volume "wordpressstarter_data" with default driver
Pulling db (mysql:5.7)...
5.7: Pulling from library/mysql
386a066cd84a: Pull complete
827c8d62b332: Pull complete
de135f87677c: Pull complete
05822f26ca6e: Pull complete
63ddbddf6165: Pull complete
15fe0fbc587e: Pull complete
93e74acdb291: Pull complete
11c2df82e984: Pull complete
d42a9e6a85c8: Pull complete
e7fb2f3afd87: Pull complete
30724006a583: Pull complete
Digest: sha256:89cc6ff6a7ac9916c3384e864fb04b8ee9415b572f872a2a4cf5b909dbbca81b
Status: Downloaded newer image for mysql:5.7
Pulling wordpress (visiblevc/wordpress:latest)...
latest: Pulling from visiblevc/wordpress
386a066cd84a: Already exists
269e95c6053a: Pull complete
6243d5c57a34: Pull complete
872f6d38a33b: Pull complete
e5ea5361568c: Pull complete
f81f18e77719: Pull complete
f9dbc878ca0c: Pull complete
195935e4100b: Pull complete
935d0c2409b2: Pull complete
d14786710093: Pull complete
b7dff268d83a: Pull complete
d1083150956d: Pull complete
9284aa2927a6: Pull complete
eb710b52f958: Pull complete
78275f047043: Pull complete
63f95fb721a4: Pull complete
488befdcaf6c: Pull complete
Digest: sha256:1e6725c90d85b6cf9b0ce0d94dd393d1a419b5486ae91c5ab2b2bf84ac87b8f3
Status: Downloaded newer image for visiblevc/wordpress:latest
Creating wordpressstarter_db_1
Creating wordpressstarter_wordpress_1
Attaching to wordpressstarter_db_1, wordpressstarter_wordpress_1
wordpress_1  | ===============================================================================
wordpress_1  |                          Begin WordPress Installation
wordpress_1  | ===============================================================================
wordpress_1  | ==> Installing WordPress
db_1         | Initializing database
db_1         | 2016-12-12T09:01:28.435611Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
db_1         | 2016-12-12T09:01:29.485261Z 0 [Warning] InnoDB: New log files created, LSN=45790
db_1         | 2016-12-12T09:01:29.782895Z 0 [Warning] InnoDB: Creating foreign key constraint system tables.
db_1         | 2016-12-12T09:01:29.946250Z 0 [Warning] No existing UUID has been found, so we assume that this is the first time that this server has been started. Generating a new UUID: 9234dc6a-c049-11e6-96f8-0242ac130003.
db_1         | 2016-12-12T09:01:29.979661Z 0 [Warning] Gtid table is not ready to be used. Table 'mysql.gtid_executed' cannot be opened.
db_1         | 2016-12-12T09:01:29.982290Z 1 [Warning] root@localhost is created with an empty password ! Please consider switching off the --initialize-insecure option.
wordpress_1  |   -> Downloading... ✓
wordpress_1  | ==> Waiting for MySQL to initialize...
db_1         | 2016-12-12T09:01:41.023523Z 1 [Warning] 'user' entry 'root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:41.023605Z 1 [Warning] 'user' entry 'mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:41.023640Z 1 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:41.023669Z 1 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:41.024356Z 1 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | Database initialized
db_1         | MySQL init process in progress...
db_1         | 2016-12-12T09:01:47.352413Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
db_1         | 2016-12-12T09:01:47.353805Z 0 [Note] mysqld (mysqld 5.7.16) starting as process 54 ...
db_1         | 2016-12-12T09:01:47.359833Z 0 [Note] InnoDB: PUNCH HOLE support available
db_1         | 2016-12-12T09:01:47.359940Z 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
db_1         | 2016-12-12T09:01:47.359950Z 0 [Note] InnoDB: Uses event mutexes
db_1         | 2016-12-12T09:01:47.359960Z 0 [Note] InnoDB: GCC builtin __atomic_thread_fence() is used for memory barrier
db_1         | 2016-12-12T09:01:47.359971Z 0 [Note] InnoDB: Compressed tables use zlib 1.2.3
db_1         | 2016-12-12T09:01:47.359977Z 0 [Note] InnoDB: Using Linux native AIO
db_1         | 2016-12-12T09:01:47.361204Z 0 [Note] InnoDB: Number of pools: 1
db_1         | 2016-12-12T09:01:47.361440Z 0 [Note] InnoDB: Using CPU crc32 instructions
db_1         | 2016-12-12T09:01:47.365674Z 0 [Note] InnoDB: Initializing buffer pool, total size = 128M, instances = 1, chunk size = 128M
db_1         | 2016-12-12T09:01:47.392392Z 0 [Note] InnoDB: Completed initialization of buffer pool
db_1         | 2016-12-12T09:01:47.397826Z 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
db_1         | 2016-12-12T09:01:47.410037Z 0 [Note] InnoDB: Highest supported file format is Barracuda.
db_1         | 2016-12-12T09:01:47.452967Z 0 [Note] InnoDB: Creating shared tablespace for temporary tables
db_1         | 2016-12-12T09:01:47.453075Z 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
db_1         | 2016-12-12T09:01:47.535896Z 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
db_1         | 2016-12-12T09:01:47.537240Z 0 [Note] InnoDB: 96 redo rollback segment(s) found. 96 redo rollback segment(s) are active.
db_1         | 2016-12-12T09:01:47.537288Z 0 [Note] InnoDB: 32 non-redo rollback segment(s) are active.
db_1         | 2016-12-12T09:01:47.537911Z 0 [Note] InnoDB: Waiting for purge to start
db_1         | 2016-12-12T09:01:47.588575Z 0 [Note] InnoDB: 5.7.16 started; log sequence number 2532794
db_1         | 2016-12-12T09:01:47.590116Z 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
db_1         | 2016-12-12T09:01:47.591006Z 0 [Note] Plugin 'FEDERATED' is disabled.
db_1         | 2016-12-12T09:01:47.595227Z 0 [Note] InnoDB: Buffer pool(s) load completed at 161212  9:01:47
db_1         | 2016-12-12T09:01:47.602466Z 0 [Warning] Failed to set up SSL because of the following SSL library error: SSL context is not usable without certificate and private key
db_1         | 2016-12-12T09:01:47.635038Z 0 [Warning] 'user' entry 'root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:47.635737Z 0 [Warning] 'user' entry 'mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:47.636318Z 0 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:47.636689Z 0 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:47.639955Z 0 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:47.648600Z 0 [Note] Event Scheduler: Loaded 0 events
db_1         | 2016-12-12T09:01:47.648937Z 0 [Note] mysqld: ready for connections.
db_1         | Version: '5.7.16'  socket: '/var/run/mysqld/mysqld.sock'  port: 0  MySQL Community Server (GPL)
db_1         | Warning: Unable to load '/usr/share/zoneinfo/iso3166.tab' as time zone. Skipping it.
db_1         | Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.
db_1         | Warning: Unable to load '/usr/share/zoneinfo/zone.tab' as time zone. Skipping it.
db_1         | 2016-12-12T09:01:53.613558Z 4 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:53.613652Z 4 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:53.613789Z 4 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 
db_1         | 2016-12-12T09:01:53.619587Z 0 [Note] Giving 0 client threads a chance to die gracefully
db_1         | 2016-12-12T09:01:53.620172Z 0 [Note] Shutting down slave threads
db_1         | 2016-12-12T09:01:53.620491Z 0 [Note] Forcefully disconnecting 0 remaining clients
db_1         | 2016-12-12T09:01:53.620779Z 0 [Note] Event Scheduler: Purging the queue. 0 events
db_1         | 2016-12-12T09:01:53.621136Z 0 [Note] Binlog end
db_1         | 2016-12-12T09:01:53.624021Z 0 [Note] Shutting down plugin 'ngram'
db_1         | 2016-12-12T09:01:53.624406Z 0 [Note] Shutting down plugin 'BLACKHOLE'
db_1         | 2016-12-12T09:01:53.624892Z 0 [Note] Shutting down plugin 'partition'
db_1         | 2016-12-12T09:01:53.625099Z 0 [Note] Shutting down plugin 'ARCHIVE'
db_1         | 2016-12-12T09:01:53.625444Z 0 [Note] Shutting down plugin 'MEMORY'
db_1         | 2016-12-12T09:01:53.625640Z 0 [Note] Shutting down plugin 'INNODB_SYS_VIRTUAL'
db_1         | 2016-12-12T09:01:53.625820Z 0 [Note] Shutting down plugin 'INNODB_SYS_DATAFILES'
db_1         | 2016-12-12T09:01:53.626009Z 0 [Note] Shutting down plugin 'INNODB_SYS_TABLESPACES'
db_1         | 2016-12-12T09:01:53.626255Z 0 [Note] Shutting down plugin 'INNODB_SYS_FOREIGN_COLS'
db_1         | 2016-12-12T09:01:53.626443Z 0 [Note] Shutting down plugin 'INNODB_SYS_FOREIGN'
db_1         | 2016-12-12T09:01:53.626581Z 0 [Note] Shutting down plugin 'INNODB_SYS_FIELDS'
db_1         | 2016-12-12T09:01:53.626814Z 0 [Note] Shutting down plugin 'INNODB_SYS_COLUMNS'
db_1         | 2016-12-12T09:01:53.626963Z 0 [Note] Shutting down plugin 'INNODB_SYS_INDEXES'
db_1         | 2016-12-12T09:01:53.627089Z 0 [Note] Shutting down plugin 'INNODB_SYS_TABLESTATS'
db_1         | 2016-12-12T09:01:53.627225Z 0 [Note] Shutting down plugin 'INNODB_SYS_TABLES'
db_1         | 2016-12-12T09:01:53.627539Z 0 [Note] Shutting down plugin 'INNODB_FT_INDEX_TABLE'
db_1         | 2016-12-12T09:01:53.627759Z 0 [Note] Shutting down plugin 'INNODB_FT_INDEX_CACHE'
db_1         | 2016-12-12T09:01:53.628229Z 0 [Note] Shutting down plugin 'INNODB_FT_CONFIG'
db_1         | 2016-12-12T09:01:53.628423Z 0 [Note] Shutting down plugin 'INNODB_FT_BEING_DELETED'
db_1         | 2016-12-12T09:01:53.628581Z 0 [Note] Shutting down plugin 'INNODB_FT_DELETED'
db_1         | 2016-12-12T09:01:53.628721Z 0 [Note] Shutting down plugin 'INNODB_FT_DEFAULT_STOPWORD'
db_1         | 2016-12-12T09:01:53.628971Z 0 [Note] Shutting down plugin 'INNODB_METRICS'
db_1         | 2016-12-12T09:01:53.629566Z 0 [Note] Shutting down plugin 'INNODB_TEMP_TABLE_INFO'
db_1         | 2016-12-12T09:01:53.631096Z 0 [Note] Shutting down plugin 'INNODB_BUFFER_POOL_STATS'
db_1         | 2016-12-12T09:01:53.631243Z 0 [Note] Shutting down plugin 'INNODB_BUFFER_PAGE_LRU'
db_1         | 2016-12-12T09:01:53.631359Z 0 [Note] Shutting down plugin 'INNODB_BUFFER_PAGE'
db_1         | 2016-12-12T09:01:53.631416Z 0 [Note] Shutting down plugin 'INNODB_CMP_PER_INDEX_RESET'
db_1         | 2016-12-12T09:01:53.631513Z 0 [Note] Shutting down plugin 'INNODB_CMP_PER_INDEX'
db_1         | 2016-12-12T09:01:53.631572Z 0 [Note] Shutting down plugin 'INNODB_CMPMEM_RESET'
db_1         | 2016-12-12T09:01:53.631674Z 0 [Note] Shutting down plugin 'INNODB_CMPMEM'
db_1         | 2016-12-12T09:01:53.631730Z 0 [Note] Shutting down plugin 'INNODB_CMP_RESET'
db_1         | 2016-12-12T09:01:53.631824Z 0 [Note] Shutting down plugin 'INNODB_CMP'
db_1         | 2016-12-12T09:01:53.631881Z 0 [Note] Shutting down plugin 'INNODB_LOCK_WAITS'
db_1         | 2016-12-12T09:01:53.632136Z 0 [Note] Shutting down plugin 'INNODB_LOCKS'
db_1         | 2016-12-12T09:01:53.632671Z 0 [Note] Shutting down plugin 'INNODB_TRX'
db_1         | 2016-12-12T09:01:53.632862Z 0 [Note] Shutting down plugin 'InnoDB'
db_1         | 2016-12-12T09:01:54.120137Z 0 [Note] InnoDB: FTS optimize thread exiting.
db_1         | 2016-12-12T09:01:54.121302Z 0 [Note] InnoDB: Starting shutdown...
db_1         | 2016-12-12T09:01:54.222529Z 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
db_1         | 2016-12-12T09:01:54.224046Z 0 [Note] InnoDB: Buffer pool(s) dump completed at 161212  9:01:54
db_1         | 2016-12-12T09:01:55.990889Z 0 [Note] InnoDB: Shutdown completed; log sequence number 12106150
db_1         | 2016-12-12T09:01:55.993461Z 0 [Note] InnoDB: Removed temporary tablespace data file: "ibtmp1"
db_1         | 2016-12-12T09:01:55.993752Z 0 [Note] Shutting down plugin 'MyISAM'
db_1         | 2016-12-12T09:01:55.993921Z 0 [Note] Shutting down plugin 'MRG_MYISAM'
db_1         | 2016-12-12T09:01:55.993968Z 0 [Note] Shutting down plugin 'CSV'
db_1         | 2016-12-12T09:01:55.993995Z 0 [Note] Shutting down plugin 'PERFORMANCE_SCHEMA'
db_1         | 2016-12-12T09:01:55.994134Z 0 [Note] Shutting down plugin 'sha256_password'
db_1         | 2016-12-12T09:01:55.994163Z 0 [Note] Shutting down plugin 'mysql_native_password'
db_1         | 2016-12-12T09:01:55.994644Z 0 [Note] Shutting down plugin 'binlog'
db_1         | 2016-12-12T09:01:55.997406Z 0 [Note] mysqld: Shutdown complete
db_1         | 
db_1         | 
db_1         | MySQL init process done. Ready for start up.
db_1         | 
db_1         | 2016-12-12T09:01:56.294984Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
db_1         | 2016-12-12T09:01:56.296105Z 0 [Note] mysqld (mysqld 5.7.16) starting as process 1 ...
db_1         | 2016-12-12T09:01:56.299685Z 0 [Note] InnoDB: PUNCH HOLE support available
db_1         | 2016-12-12T09:01:56.299728Z 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
db_1         | 2016-12-12T09:01:56.299740Z 0 [Note] InnoDB: Uses event mutexes
db_1         | 2016-12-12T09:01:56.299747Z 0 [Note] InnoDB: GCC builtin __atomic_thread_fence() is used for memory barrier
db_1         | 2016-12-12T09:01:56.299753Z 0 [Note] InnoDB: Compressed tables use zlib 1.2.3
db_1         | 2016-12-12T09:01:56.299758Z 0 [Note] InnoDB: Using Linux native AIO
db_1         | 2016-12-12T09:01:56.300302Z 0 [Note] InnoDB: Number of pools: 1
db_1         | 2016-12-12T09:01:56.300654Z 0 [Note] InnoDB: Using CPU crc32 instructions
db_1         | 2016-12-12T09:01:56.302816Z 0 [Note] InnoDB: Initializing buffer pool, total size = 128M, instances = 1, chunk size = 128M
db_1         | 2016-12-12T09:01:56.315674Z 0 [Note] InnoDB: Completed initialization of buffer pool
db_1         | 2016-12-12T09:01:56.318584Z 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
db_1         | 2016-12-12T09:01:56.373396Z 0 [Note] InnoDB: Highest supported file format is Barracuda.
db_1         | 2016-12-12T09:01:56.494724Z 0 [Note] InnoDB: Creating shared tablespace for temporary tables
db_1         | 2016-12-12T09:01:56.494852Z 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
db_1         | 2016-12-12T09:01:56.596765Z 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
db_1         | 2016-12-12T09:01:56.599294Z 0 [Note] InnoDB: 96 redo rollback segment(s) found. 96 redo rollback segment(s) are active.
db_1         | 2016-12-12T09:01:56.599357Z 0 [Note] InnoDB: 32 non-redo rollback segment(s) are active.
db_1         | 2016-12-12T09:01:56.600252Z 0 [Note] InnoDB: Waiting for purge to start
db_1         | 2016-12-12T09:01:56.652876Z 0 [Note] InnoDB: 5.7.16 started; log sequence number 12106150
db_1         | 2016-12-12T09:01:56.655143Z 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
db_1         | 2016-12-12T09:01:56.655663Z 0 [Note] Plugin 'FEDERATED' is disabled.
db_1         | 2016-12-12T09:01:56.666452Z 0 [Warning] Failed to set up SSL because of the following SSL library error: SSL context is not usable without certificate and private key
db_1         | 2016-12-12T09:01:56.666453Z 0 [Note] InnoDB: Buffer pool(s) load completed at 161212  9:01:56
db_1         | 2016-12-12T09:01:56.668618Z 0 [Note] Server hostname (bind-address): '*'; port: 3306
db_1         | 2016-12-12T09:01:56.669202Z 0 [Note] IPv6 is available.
db_1         | 2016-12-12T09:01:56.669555Z 0 [Note]   - '::' resolves to '::';
db_1         | 2016-12-12T09:01:56.669746Z 0 [Note] Server socket created on IP: '::'.
db_1         | 2016-12-12T09:01:56.701390Z 0 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:56.701883Z 0 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:56.710872Z 0 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
db_1         | 2016-12-12T09:01:56.717366Z 0 [Note] Event Scheduler: Loaded 0 events
db_1         | 2016-12-12T09:01:56.717693Z 0 [Note] mysqld: ready for connections.
db_1         | Version: '5.7.16'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
wordpress_1  |   ->  mysqld is alive
wordpress_1  | ==> Configuring WordPress
wordpress_1  |   -> Generating wp-config.php file... ✓
wordpress_1  | ==> Checking database
wordpress_1  |   -> Creating database mywebsite ✓
wordpress_1  |   -> No database backup found. Initializing new database ✓
```

### Import output through wp cli
I did a npm install at the root of the folder before running the command. 

```
MacBook-Air-de-Laila:wordpress-starter leiluspocus$ npm run wp db import /data/database.sql

> visible-wordpress@0.12.1 wp /Users/lailaatrmouh/Work/wordpress-starter
> wpcmd() { docker exec $(docker ps -lq) /bin/bash -c "sudo -u www-data wp $(echo $@)"; };wpcmd  "db" "import" "/data/database.sql"

/bin/bash: sudo: command not found

npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "wp" "db" "import" "/data/database.sql"
npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! visible-wordpress@0.12.1 wp: `wpcmd() { docker exec $(docker ps -lq) /bin/bash -c "sudo -u www-data wp $(echo $@)"; };wpcmd  "db" "import" "/data/database.sql"`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the visible-wordpress@0.12.1 wp script 'wpcmd() { docker exec $(docker ps -lq) /bin/bash -c "sudo -u www-data wp $(echo $@)"; };wpcmd  "db" "import" "/data/database.sql"'.
npm ERR! This is most likely a problem with the visible-wordpress package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     wpcmd() { docker exec $(docker ps -lq) /bin/bash -c "sudo -u www-data wp $(echo $@)"; };wpcmd  "db" "import" "/data/database.sql"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs visible-wordpress
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls visible-wordpress
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lailaatrmouh/Work/wordpress-starter/npm-debug.log
```

Please let me know if you need any more information.

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -6,7 +6,7 @@ services:
       - 8080:80
       - 443:443
     volumes:
-      - data:/data
+      - ./data:/data
     environment:
       DB_NAME: wordpress
       DB_PASS: root

